docs/cli/edgedb_branch/edgedb_branch_create.rst

```diff
@@ -0,0 +1,48 @@
+.. _ref_cli_edgedb_branch_create:
+
+
+====================
+edgedb branch create
+====================
+
+.. note::
+
+    This CLI command requires CLI version 4.0 or later and EdgeDB version 5.0
```

Comment:
Meant to ask you about this specifically, @quinchs: what will the version be on the CLI release that contains branching? Is 4.0 correct?

Reply:
I think 4.3.0